week16/1034_램프_eunhxx.py

```diff
@@ -0,0 +1,26 @@
+N,M=map(int, input().split()) #N:행, M:열
+lamp=[list(map(int, input())) for _ in range(N)] #램프상태 저장
+K = int(input()) #0~1000
+tmp, result=[], 0
+
+
+for i in range(N):
+    cnt0 = 0 #0갯수 세기위한 변수
+    cnt_row = 0 #동일한 행 개수 세기 위한 변수
+    
+    for j in range(M):
+        if lamp[i][j] == 0:
+            cnt0+=1 #0갯수 카운트
```

Comment:
이거 그냥
```suggestion
cnt0=lamp[i].count(0)
```
이렇게 했어도 되었을듯? 근데 어차피 전체 배열봐야해서 시간복잡도는 같아 ㅋㅋㅋㅋ

Reply:
그러게 훨씬 간단해보임!! count함수가 있는지 몰랐닼ㅋㅋㅋ